content/acra/configuring-maintaining/controls-configuration-on-acratranslator/_index.md

```diff
@@ -34,8 +34,8 @@ AcraTranslator can be configured from command line using CLI flags ([see the who
 
   For TLS, there are a plenty of
   [`--tls_*` options](/acra/configuring-maintaining/general-configuration/acra-translator/#tls).
-  You can use either TLS (for direct connections from clients) or Themis Secure Session (when using AcraConnector).
+  You can use either TLS (for direct connections from clients).
```

Comment:
I think it's better to leave this line out. Or at least remove the `either` and parentheses.